### PR TITLE
[April 27, 2023 9AM PST] dbt Cloud GA for Starburst/Trino adapter support 

### PR DESCRIPTION
**Pull request by nghi-ly**
## What are you changing in this pull request and why?
To support the GA of the Starburst/Trino adapter in dbt Cloud, remove the public preview calllout and update prerequisites in the quickstart: 

https://docs.getdbt.com/docs/quickstarts/dbt-cloud/starburst-galaxy

## Checklist
- [x] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) and [About versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) so my content adheres to these guidelines.
- [ ] **DO NOT MERGE** this PR until launch day

### CODE REVIEW

File: website/docs/docs/quickstarts/dbt-cloud/starburst-galaxy-qs.md [hunk 1]
```diff
@@ -17,13 +17,8 @@ In this quickstart guide, you'll learn how to use dbt Cloud with [Starburst Gala
...
 - Connect to multiple data sources in addition to your S3 bucket. 
 
-:::tip Public Preview 
-The Starburst (Trino compatible) connection in dbt Cloud is currently available in Public Preview for multi-tenant dbt Cloud accounts. Preview features are stable and can be considered for production deployments, but there might still be some planned additions and modifications to product behavior before moving to General Availability. For details, refer to [dbt Product lifecycles](/docs/dbt-versions/product-lifecycles). 
-:::
-
-
 ## Prerequisites 
-- You have a [dbt Cloud account](https://www.getdbt.com/signup/).
+- You have [multi-tenant](/docs/cloud/about-cloud/regions-ip-addresses) deployment in [dbt Cloud](https://www.getdbt.com/signup/). For more information, refer to [Tenancy](/docs/cloud/about-cloud/tenancy).
```
Comment (runleonarun) [inline website/docs/docs/quickstarts/dbt-cloud/starburst-galaxy-qs.md]:
```suggestion
- You have a [multi-tenant](/docs/cloud/about-cloud/regions-ip-addresses) deployment in [dbt Cloud](https://www.getdbt.com/signup/). For more information, refer to [Tenancy](/docs/cloud/about-cloud/tenancy).
```

**Review by runleonarun — DISMISSED** [review]
Approved for GA on Thursday.

**Comment by mirnawong1** [comment]
hey @runleonarun my two pr's are merged and you should now be able to see the full pr for this - thank yoU!

**Review by runleonarun — APPROVED** [review]
This looks ready to 🚢! Approving for tomorrow's GA

**Pull request merged**